internal/payloads/null_convert.go

```diff
@@ -0,0 +1,11 @@
+package payloads
+
+import "database/sql"
+
+func NullConvert(s sql.NullString) *string {
```

Comment:
Perhaps this is more future-proof:

```suggestion
func SqlNullToStringPtr(s sql.NullString) *string {
```

We will be likely adding few more.